content/desktop/release-notes.md

```diff
@@ -21,11 +21,71 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.31.0
+
+{{< release-date date="2024-05-30" >}}
+
+{{< desktop-install all=true beta_win_arm=true version="4.31.0" build_path="/TODO_NOT_AVAILABLE_YET/" >}}
+
+### New
+
+- Admins can now use alternative methods to the `registry.json` file, to enforce sign in.
+- [Air-Gapped Containers](desktop/hardened-desktop/air-gapped-containers.md) is now generally available. 
+- Introducing: compose bridge. You might be asked your password to install it. Make sure in settings->advanced->Automatically check configuration is turned on.
+- Added Multi-node Kind as a Kubernetes option (Beta).
```

Comment:
```suggestion
```
Remove this line as well. Unfortunately, @milas's departure from our team has delayed our multi-node Kind release for DD. We're still scoping the updated timeline given impacts to our team velocity.